crates/universal-sierra-compiler-api/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::Context;
+use std::path::Path;
+use std::str::from_utf8;
+
+pub use command::*;
+
+mod command;
+
+#[must_use]
+pub fn compile_sierra(sierra_file_name: &str, current_dir: Option<&Path>) -> String {
+    let mut usc_command = UniversalSierraCompilerCommand::new();
+    if let Some(dir) = current_dir {
+        usc_command.current_dir(dir);
+    }
+
+    let usc_output = usc_command
+        .inherit_stderr()
+        .arg("--sierra-input-path")
+        .arg(sierra_file_name)
+        .command()
+        .output()
+        .context(
+            "Error while compiling Sierra of the contract. \
+            Make sure you have the latest universal-sierra-binary installed. \
+            Contact us if it doesn't help",
+        )
+        .unwrap();
```

Comment:
I don't think we should unwrap, it may be users fault, we should pretty print that - `?` and return `Result` from this function

Reply:
Who is the `user`?

Reply:
Someone using snforge, having older version of our compiler installed